src/bin/vip.js

```diff
@@ -55,17 +55,17 @@ const rootCmd = async function() {
 		try {
 			token = new Token( t );
 		} catch ( e ) {
-			console.log( 'Invalid token: misformed' );
+			console.log( 'The token provided is malformed. Please check the token and try again.' );
 			return;
 		}
 
 		if ( token.expired() ) {
-			console.log( 'Invalid token: expired' );
+			console.log( 'The token provided is expired. Please log in again to refresh the token.' );
```

Comment:
Generally this error is going to happen when you're trying to log into the CLI. Should we specify that you have to go back to the admin to get a valid token?

Reply:
The `vip login` flow already you to the Dashboard and they need to run it again to insert the token. Going to deploy for now, but if we have ideas on a clearer message + path for this flow, we can update.

I was also thinking that we may want to add a catch-all statement to contact VIP Support for these token failures since something is probably broken if it keeps happening and they probably need more help.